### DISCUSSION TITLE
Directly past

### DISCUSSION THREAD

**Discussion by tahadnan** [fdw/rofimoji#189] [Q&A] [answered]
Hello guys, forst I want to thank the team behind this great project, my question is is there a way to past the emoji directly in the current writing zone, like bringing up the selector choose the emoji, click enter, and boom it's pasted.

**Comment by fdw** [comment, marked as answer]
That depends a bit: If you use the standalone mode, then the `type` action should do exactly that, and it is the default. However, you need a compatible "typer" and especially with `xdotool`, there are sometimes problems.
If you use it as a rofi mode, then directly inserting the emoji is unfortunately not possible.

Also have a look at [Insertion Method](https://github.com/fdw/rofimoji?tab=readme-ov-file#insertion-method) and [the caveat for the rofi mode](https://github.com/fdw/rofimoji?tab=readme-ov-file#caveats-when-running-rofimoji-as-a-rofi-mode).
